src/main/java/com/github/kongchen/swagger/docgen/SwaggerModule2.java

```diff
@@ -0,0 +1,17 @@
+package com.github.kongchen.swagger.docgen;
+
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+public class SwaggerModule2 extends SimpleModule {
```

Comment:
This class needs a better name than just adding "2". Please make it descriptive of functionality.